docker-vms/nginx/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM tutum/nginx
```

Comment:
I've actually swopped this out now as I was working on something else and realised I could use a different one which auto discovers them. See the last commit and details: https://github.com/jwilder/nginx-proxy